aws/eks/monitoring.tf

```diff
@@ -1,7 +1,27 @@
-
 module "prometheus" {
-  source                   = "github.com/mozilla-it/terraform-modules//helm/prometheus?ref=master"
+  source                   = "../../helm/prometheus"
   enabled                  = local.cluster_features["prometheus"]
   cloud_provider           = "aws"
   prometheus_helm_settings = var.prometheus_settings
+  influxdb                 = var.influxdb
+}
+
+resource "helm_release" "prometheus_customizations" {
+  # only checking dependencies here, not features.  So maybe these customizations can be used in a modular way going forward.
```

Comment:
appreciate this comment, was thinking something similar when reading through this